atomwrappers.py

```diff
@@ -59,15 +59,75 @@ def recursive_print(self, result='', indent=''):
 class CEvaluationLink(CLink):
     atom_type = 'EvaluationLink'
 
+class CExecutionLink(CLink):
+    atom_type = 'ExecutionLink'
+
+class CLazyExecutionOutputLink(CLink):
+    atom_type = 'LazyExecutionOutputLink'
+
+class CQuantitativePredicateLink(CLink):
+    atom_type = 'QuantitativePredicateLink'
+
 class CPredicateNode(CNode):
     atom_type = 'PredicateNode'
 
+class CSchemaNode(CNode):
+    atom_type = 'SchemaNode'
+
+class CQuantitativeSchemaNode(CNode):
+    atom_type = 'QuantitativeSchemaNode'
+
+class CQuantitativePredicateNode(CNode):
+    atom_type = 'QuantitativePredicateNode'
+
 class CConceptNode(CNode):
     atom_type = 'ConceptNode'
 
+class CNumberNode(CNode):
+    atom_type = 'NumberNode'
+
 class CMoleculeNode(CNode):
     atom_type = 'MoleculeNode'
 
+class CChebiNode(CNode):
+    atom_type = 'ChebiNode'
+
+class CUniprotNode(CNode):
+    atom_type = 'UniprotNode'
+
+class CRnaNode(CNode):
+    atom_type = 'RnaNode'
+
+class CPubchemNode(CNode):
+    atom_type = 'PubchemNode'
+
+class CCodingRna(CNode):
```

Comment:
We don't have coding and non coding RNA's as a type rather, they are Refseq and Enst (ensemble transcripts).